### DISCUSSION TITLE
第6节课作业(7班)

### DISCUSSION THREAD

**Discussion by JimmyMa99** [InternLM/Tutorial#461] [作业]
第6节课作业(7班)

**Comment by wuhaostudio** [comment]
基础作业:internlm 
<img width="1028" alt="微信图片_20240121114400" src="https://github.com/InternLM/tutorial/assets/143194075/798221cc-a702-4b38-8c47-189af7a065ac">

**Reply by wuhaostudio** [reply]
InternLM2
<img width="1080" alt="微信图片_20240122095732" src="https://github.com/InternLM/tutorial/assets/143194075/f9bb671f-ae56-494f-b553-5dd16ff5bfc9">

**Comment by SaaRaaS-1300** [comment]
第6节课，笔记+作业 Github：https://github.com/SaaRaaS-1300/InternLM_openNotebook/blob/main/Lesson-6/Lesson-6-Notebook.md

**Comment by createup2023** [comment]
https://www.yuque.com/creatup/ynrsde/dqi0aa8uir28rbax?singleDoc# 《第六次作业：》
![image](https://github.com/InternLM/tutorial/assets/128460366/ece36c28-332b-4f97-8d47-b96f6f9784b0)

**Comment by PommesPeter** [comment]
![image](https://github.com/InternLM/tutorial/assets/54879512/3b62415f-b6e2-4d15-b1c8-4ad639d250a8)

**Comment by JimmyMa99** [comment]
https://github.com/JimmyMa99/internlmtutorial/blob/main/%E5%9F%BA%E7%A1%80%E4%BD%9C%E4%B8%9A%E5%90%88%E9%9B%86.md

**Comment by lifelinezhang** [comment]
https://zhuanlan.zhihu.com/p/679285190

**Comment by anneeeee** [comment]
<img width="1012" alt="image" src="https://github.com/InternLM/tutorial/assets/16276686/b33ba04a-5ea8-4e76-8eb4-42a23ceb2f7f">

**Comment by hui1feng** [comment]
第六节课作业：
![捕获](https://github.com/InternLM/tutorial/assets/126125104/a5d44fee-7e67-4184-b3fc-f5e5be769b10)
![捕获0](https://github.com/InternLM/tutorial/assets/126125104/3a0445b7-4610-4014-be8d-4ddc52fc596d)

**Comment by abelezm** [comment]
第六课基础作业：https://blog.csdn.net/weixin_38043453/article/details/135820046


![image](https://github.com/InternLM/tutorial/assets/92080418/2e2d2d0e-24d5-4037-ad02-55522bf1e428)

**Comment by leochiang** [comment]
<img width="868" alt="71b0e546a335c62effcf239ff9aae46" src="https://github.com/InternLM/tutorial/assets/7628117/10244b51-ca14-4e8c-96a9-f4e7067802bc">

**Comment by Rogue-Trader-zzy** [comment]
作业：
![image](https://github.com/InternLM/tutorial/assets/64819379/1e42711b-1164-4607-a24b-5c463ca2a63c)
![image](https://github.com/InternLM/tutorial/assets/64819379/10fbb5dc-dc13-4c87-9a19-9a51d8c252a0)
后面再详细补一下。

**Comment by Simon-lyz** [comment]
## InternLM Lesson-6 Opencompass评测
https://g4w90egc30.feishu.cn/docx/ZOJZdv1duotLW4x0Nv8cWpyVnEb?from=from_copylink
